sim/ofat.py

```diff
@@ -121,6 +126,31 @@ def analyse(root_path):
         "tp": tps
     })
 
+def plot(root_path):
+
+    dirs = glob.glob(root_path + "/*/stats.csv")
```

Comment:
Validate that root_path exists? There's a function for this in sim/utils.py "assert_path_exists(path):"